Assets/MixedRealityToolkit/Interfaces/Services/IMixedRealityDataProviderAccess.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.MixedReality.Toolkit
+{
+    public interface IMixedRealityDataProviderAccess
+    {
+        /// <summary>
+        /// Gets the collection of registered data providers.
+        /// </summary>
+        /// <returns>
+        /// Read only copy of the list of registered observers.
```

Comment:
>observers [](start = 53, length = 9)

Should this be "observer" or data provider? I know that the word observer has a different meaning in the spatial awareness system world, and I haven't seen this referenced in the other input parts before.

Reply:
```suggestion
        /// Read only copy of the list of registered data providers.
```

Reply:
Same comment for the function down below as well.

---
In reply to: [285701349](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/4466#discussion_r285701349) [](ancestors = 285701349)